src/css/application.css

```diff
@@ -131,67 +133,87 @@ body {
   background-color: var(--color-red);
 }
 
-.top-bar__logo-container > svg {
-  height: 3em;
+.top-bar__logo-container {
+  margin-right: 0.2em;
+  min-width: 1.6em;
+  transform: scale(1.4);
+  width: 1.6em;
 }
 
 .top-bar__wordmark-container {
   position: relative;
-  margin-left: -0.25em;
+  margin: 0 1em 0 0.5em;
+  min-width: 6.5em;
+  width: 6.5em;
 }
 
 .top-bar__wordmark-container > svg {
-  padding: 0 1em;
-  max-height: 1em;
-  max-width: 100%;
-  width: auto;
+  width: 100%;
+  height: 100%;
 }
 
 .top-bar__snapshot_in-progress {
-  color: var(--color-light-gray);
+  opacity: 0.5;
   cursor: progress;
 }
 
 .top-bar__avatar {
-  height: 2em;
-  border-radius: 50%;
```

Comment:
Hmm, honestly I think the circle looks quite a bit nicer…

Reply:
I agree the circle is trendy and looks good, but 2 reasons I changed it. 1, Github uses squares and 2, it matches the other buttons and general style. Happy to go back to circle if you really feel strongly about it though.

Reply:
I’m into the way the rounded corners look.